ecommerce_integrations/shopify/fulfillment.py

```diff
@@ -72,3 +72,36 @@ def get_fulfillment_items(dn_items, fulfillment_items, location_id=None):
 		for dn_item in dn_items
 		if get_item_code(item) == dn_item.item_code
 	]
+
+
+def update_fulfillment_status(payload, request_id=None):
```

Comment:
The reason why we didn't do this in first place is there are many ways people handle returns and refunds. 

If you add a very specific behaviour that too isn't configurable then it's gonna annoy a a huge amount of users. 

Please make this configurable as much as possible with some rationale behind each option. 


Defaults should be - everything disabled like status quo.

Reply:
Have added the config:
![image](https://user-images.githubusercontent.com/52111700/216629067-6608040b-e237-466f-8992-a1ee2a7578a5.png)